Library/Homebrew/cmd/doctor.rb

```diff
@@ -435,6 +443,7 @@ def check_access_usr_local
 
     You should probably change the ownership and permissions of /usr/local
     back to your user account.
+      sudo chown $(whoami):admin /usr/local
```

Comment:
Can `/usr/local` itself still be owned by `root:wheel` and be completely usable if `brew` owns everything in it?